bin/ch/ch.cpp

```diff
@@ -129,8 +128,8 @@ void __stdcall PrintChakraCoreVersion()
                 // Doesn't matter if you are on 32 bit or 64 bit,
                 // DWORD is always 32 bits, so first two revision numbers
                 // come from dwFileVersionMS, last two come from dwFileVersionLS
-                printf("%s version %d.%d.%d.%d\n",
-                    chakraDllName,
+                wprintf(_u("%s version %d.%d.%d.%d\n"),
+                    GetChakraDllNameW(),
```

Comment:
>GetChakraDllNameW(), [](start = 20, length = 20)

Wonder if you can use format specifier "%S" with narrow chakraDllName. Save creating a "W" version.